Cargo.toml

```diff
@@ -13,7 +13,7 @@ name = "bindgen"
 readme = "README.md"
 repository = "https://github.com/rust-lang-nursery/rust-bindgen"
 documentation = "https://docs.rs/bindgen"
-version = "0.32.3"
+version = "0.32.4"
```

Comment:
This version bump needs to be major, since breaking changes have landed since the last release.

Reply:
Oh, sorry, I didn't check that ^^